### ISSUE TITLE
Dem User es einfacher machen, bei Problemen mit uns Kontakt aufzunehmen

### ISSUE THREAD

**Issue by xeruf** [CAU-Kiel-Tech-Inf/socha-gui#3]
From socha_graphical_client created by [SKoschnicke](https://github.com/SKoschnicke) : CAU-Kiel-Tech-Inf/socha_graphical_client#98  

Z.B. ein Crash Reporter oder auch die Moeglichkeit, uns direkt ein App-Log und/oder Replay zukommen zu lassen, mit einer Nachricht dazu.

Normalerweise laeuft das im Moment so, dass wir eine Mail bekommen mit "Das und das funktioniert nicht". Dann fragen wir nach, welches Betriebssystem, welche App Version, eventuell ein Replay oder Log. Und dann koennen wir erst wirklich helfen.

In allgemeiner Form mit Stack-Traces gibt es da schon was fertiges: https://xwartz.gitbooks.io/electron-gitbook/content/en/api/crash-reporter.html, aber ich moechte in jedem Fall den User fragen, bevor irgendwas uebermittelt wird.

**Comment by SKoschnicke** [comment]
Da haben wir bald einen Studenten der sich im Rahmen einer Abschlussarbeit mit beschaeftigt.

**Comment by xeruf** [comment]
Abschlussarbeit? Über error reporting? :sweat_smile:

**Comment by SKoschnicke** [comment]
Nein, ein allgemeines Feedback-System, welches überall verfügbar ist und eine möglichst geringe Schwelle aufweist, mit uns in Kontakt zu treten.
